cmds/core/date/date_windows.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2015-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"os"
+	"time"
+)
+
+func setDate(d string, z *time.Location, clocksource Clock) error {
+	return os.ErrPermission
```

Comment:
Is there a reason you chose `os.ErrPermission?` Maybe something like `os.ErrNotExist` fits better for something not implemented? Further, let's choose the same error in file [`date_plan9.go`](https://github.com/u-root/u-root/blob/17eba74d7b54563edc30bf206b4ff65d4c557f8f/cmds/core/date/date_plan9.go) instead of a custom error?

Reply:
note that plan 9 errors are strings, but point taken.

Reply:
ok, I think it's right now. Thanks.